packages/gestalt-charts/src/ChartGraph/TabularDataModalTable.js

```diff
@@ -0,0 +1,76 @@
+// @flow strict-local
+import { type Node } from 'react';
+import { Table, Text, useDefaultLabel } from 'gestalt';
+import { type TransformedTabularDataType } from './useTabularData.js';
+
+type Props = {|
+  title: string,
+  isHorizontalLayout: boolean,
+  transformedTabularData: TransformedTabularDataType,
+  onSortChange: ('series' | 'x' | 'y') => void,
+  sortOrder: 'desc' | 'asc',
+  sortCol: null | 'series' | 'x' | 'y',
```

Comment:
Can we extract these to union type declaration? They have been repeated a lot.

Reply:
DONE